app/src/main/kotlin/com/simplemobiletools/dialer/activities/DialpadActivity.kt

```diff
@@ -31,6 +32,7 @@ class DialpadActivity : SimpleActivity() {
     private val russianCharsMap = HashMap<Char, Int>()
     private var hasRussianLocale = false
     private var privateCursor: Cursor? = null
+    private var toneGeneratorHelper: ToneGeneratorHelper? = null
```

Comment:
this is always null, beeps dont work

Reply:
[fixed](https://github.com/SimpleMobileTools/Simple-Dialer/pull/436/commits/60ea4ec0f354797805c79ad6be4b65074b90b0d7)